src/Nethermind/Nethermind.Db/LogIndex/ILogIndexStorage.cs

```diff
@@ -0,0 +1,30 @@
+﻿// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+
+namespace Nethermind.Db;
+
+public readonly record struct BlockReceipts(int BlockNumber, TxReceipt[] Receipts);
+
+public interface ILogIndexStorage : IAsyncDisposable
+{
+    int GetLastKnownBlockNumber();
+    IEnumerable<int> GetBlockNumbersFor(Address address, int from, int to);
+
+    IEnumerable<int> GetBlockNumbersFor(Hash256 topic, int from, int to);
```

Comment:
Topic does not include the index. What if I'm searching for transfer from or to an account 
`Transfer (index_topic_1 address src, index_topic_2 address dst, uint256 wad)`. Will I get the same data and potentially some false positives?

Reply:
Yes, false-positives are possible.

Reply:
- For randomly generated 20k blocks - `175.48 MB` vs `206.92 MB` (`+18%`).
- For 1M blocks (19M to 20M) from Mainnet - `1.71 GB` vs `1.92 GB` (`+12%`)

Very rough estimate would be `5 - 10%` size increase for the whole Mainnet.